src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import styles from "./SearchBar.module.css";
+
+type SearchBarProps = {
+  onSearch: (value: string) => void;
+};
+
+function SearchBar({ onSearch }: SearchBarProps): JSX.Element {
+  const [value, setValue] = useState("");
+
+  console.log(value);
+  console.log(onSearch);
```

Comment:
```suggestion
```

Reply:
Would result in an error and I’ll need them later.